### PR TITLE
Include Hotfix to change Truck logit exp to -4

### PR DESCRIPTION
**Pull request by johannah-pik**
Fix the issue, that Trucks get a placeholder logit exponent of -10 due to missing Truck types in GCAM

### CODE REVIEW

**Review by MariannaR — COMMENTED** [review]
Light trucks and SUVs are under "LDVs", maybe you can include a check also on the subsector_L1 level to be sure they are not affected

**Comment by johannah-pik** [comment]
> Light trucks and SUVs are under "LDVs", maybe you can include a check also on the subsector_L1 level to be sure they are not affected

That is true, thanks! I will include and update it!

**Comment by johannah-pik** [comment]
> > Light trucks and SUVs are under "LDVs", maybe you can include a check also on the subsector_L1 level to be sure they are not affected
> 
> That is true, thanks! I will include and update it!

I double-checked: "Light Truck and SUV" are not included here as only "dfother" is updated for all vehicle types that receive shareweights instead of inconveniece costs

File: R/calculate_logit_inconv_endog.R [hunk 1]
```diff
@@ -385,6 +386,10 @@ calculate_logit_inconv_endog = function(prices,
     ## overwrite the preferences with the market based ones
     pref_data[["FV_final_pref"]] = final_prefFV
 
+    ##HOTfix: Truck size classes are not all included in the input data -> they need to be added manually
+    ##ATTENTION: This should be changed in the refactoring process
+    dfother[grepl("Truck", vehicle_type), logit.exponent := -4]
```
Comment (Loisel) [inline R/calculate_logit_inconv_endog.R:391]:
```suggestion
    dfother[grepl("^Truck", vehicle_type), logit.exponent := -4]
```

Comment (Loisel) [reply R/calculate_logit_inconv_endog.R:391]: In this case I agree with Marianna that explicitly including only vehicle_types that start with `Trucks` is less ambiguous although technically you are right that they are not affected.

**Pull request merged**